client/components/status/status.js

```diff
@@ -276,6 +279,15 @@ export default class Status extends React.Component {
     this.setState({screenWidth: window.innerWidth});
   }
 
+  updateSpinner = () => {
+    const {activeSessions, passedSessions} = this.state;
```

Comment:
I didn't notice this before, `passedSessions` doesn't sounds like a good name to me, what about `pastSessions`? Please change all the occurrences.